Sources/AppcuesKit/Presentation/Traits/Appcues/AppcuesModalTrait.swift

```diff
@@ -98,3 +113,40 @@ extension AppcuesModalTrait {
         }
     }
 }
+
+@available(iOS 13.0, *)
+extension AppcuesModalTrait.Config {
+    func toTransition() -> AppcuesModalTrait.Transition {
+
+        switch self.transition {
+        case "slide":
+            // determine the slide edge based on the horizontal and vertical
+            // position of the modal
+            var edge: AppcuesModalTrait.TransitionEdge
+
+            let horizontalAlign = style?.horizontalAlignment ?? "center"
+            let verticalAlign = style?.verticalAlignment ?? "center"
+
+            switch horizontalAlign {
+            case "leading":
+                edge = .leading
+            case "trailing":
+                edge = .trailing
+            default:
+                switch verticalAlign {
+                case "top":
+                    edge = .top
+                case "bottom":
+                    edge = .bottom
+                default:
+                    edge = .center
+                }
+            }
```

Comment:
Nit, but I'm not a big fan of overly nested switches. Maybe this if it looks good to you?
```swift
switch (horizontalAlign, verticalAlign) {
case ("leading", _):
    edge = .leading
case ("trailing", _):
    edge = .trailing
case (_, "top"):
    edge = .top
case (_, "bottom"):
    edge = .bottom
default:
    edge = .center
}
```

Reply:
yeah thats much cleaner, thanks